diff --git a/current/core/server/models/post.orig.js b/current/core/server/models/post.js
index 4576534..7b848d5 100644
--- a/current/core/server/models/post.orig.js
+++ b/current/core/server/models/post.js
@@ -15,6 +15,9 @@ var _               = require('lodash'),
     Post,
     Posts;
 
+var ghostAlgolia    = require(config.getContentPath('apps') + 'ghost-algolia');
+ghostAlgolia.registerEvents(events);
+
 Post = ghostBookshelf.Model.extend({
 
     tableName: 'posts',
